src/libraries/icubmod/imu3DM_GX3/CMakeLists.txt

```diff
@@ -14,4 +14,11 @@ if (ENABLE_${PROJECTNAME})
     yarp_add_plugin(${PROJECTNAME} 3dm_gx3.cpp 3dm_gx3.h )
     target_link_libraries(${PROJECTNAME} ${YARP_LIBRARIES})
     icub_export_plugin(${PROJECTNAME})
+
+  yarp_install(TARGETS ${PROJECTNAME}
```

Comment:
@barbalberto  can you check this for me? It seems the yarp_install was completely missing...

Reply:
It was used on Coman/Walkman robots when icubmod was compiled statically.
I think they have imported inthey repositories, anyway this change it is safe, it is fine with me

Reply:
👍